### DISCUSSION TITLE
Displaying articles updated as new

### DISCUSSION THREAD

**Discussion by baxi10** [tughi/aggregator-android#78] [Ideas]
When only new articles are displayed in the application, the application does not display articles that have been updated. I need an option that allowsdisplay udpated read articles as new.

**Comment by tughi** [comment]
Wouldn't this cause too much noise? I follow many feeds that update the same entry only because of typos, and I wouldn't want to read them again.

**Reply by baxi10** [reply]
You're right, I thought about that later. However, I follow a lot of forums with RSS feeds that contain movie series. If there is a new episode in a thread, I can't see it if "Show read entries" is turned off. Maybe a better solution would be to display updated posts only in tags? Maybe as an additional option in the conditions of the tag?
I don't know if I described it well.
